index.html

```diff
@@ -1430,9 +1441,50 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
```

Comment:
We have to mention non-white races if we list groups, and we'll probably get in trouble for missing other disadvantaged groups. How about:

```suggestion
particularly via social media. Harassment may be more severe and its consequences more impactful for
people who are disadvantaged in society.
```